h/conftest.py

```diff
@@ -91,6 +92,8 @@ def db_session(request, settings):
     engine = db.make_engine(settings)
     db.bind_engine(engine, should_create=True, should_drop=True)
 
+    api_db.use_session(db.Session)
```

Comment:
This isn't necessary. You're already calling `db.make_engine` three lines earlier, and that calls this, no?